tests/business.test.ts

```diff
@@ -10,20 +10,58 @@ import { Marqeta } from '../src';
   })
 
   console.log('getting list of Businesses...')
-  let one
-  one = await client.business.list()
+  const one = await client.business.list()
   if (one?.success && one.body?.isMore) {
     console.log(`Success! A list ${one?.body!.count} Businesses were retrieved: `)
-  } else {
+  }
+  else {
     console.log('Error! Unable to get a list of Businesses.')
     console.log(one)
   }
   console.log('getting list of unknown Businesses with search parameter: "{ businessNameDba:xyz }"...')
-  one = await client.business.list({ businessNameDba:'xyz' })
-  if (one?.success && one.body?.isMore) {
-    console.log(`Success! A list ${one?.body!.count} Businesses were retrieved: `)
-  } else {
+  const twoA = await client.business.list({ businessNameDba:'xyz' })
+  if (twoA?.success && twoA.body?.isMore) {
+    console.log(`Success! A list ${twoA?.body!.count} Businesses were retrieved: `)
+  }
+  else {
     console.log('Error! Unable to get a list of Businesses.')
-    console.log(one)
+    console.log(twoA)
+  }
+  
+  console.log('creating a Business account...')
+  const twoB = await client.business.create({
+    business: {
```

Comment:
Yeah... with my suggestion this could look much cleaner - Right?

Reply:
much cleaner